test/minio.e2e.test.ts

```diff
@@ -1,103 +1,473 @@
-import { afterAll, beforeAll, describe, expect, test } from "bun:test";
 import { randomUUID } from "node:crypto";
+import fs from "node:fs/promises";
+import { tmpdir } from "node:os";
 import { basename, join } from "node:path";
-import { createServerMinioClient } from "@settlemint/sdk-minio";
-import { loadEnv } from "@settlemint/sdk-utils/environment";
-import type { Client } from "minio";
-
-const TEST_BUCKET_NAME = `test-bucket-${randomUUID()}`;
-const TEST_FILES = [join(__dirname, "./data/test-pdf.pdf"), join(__dirname, "./data/test-image.png")];
-
-describe("MinIO E2E Tests", () => {
-  let client: Client;
-
-  beforeAll(async () => {
-    const env = await loadEnv(true, false);
-    const { client: minioClient } = createServerMinioClient({
-      instance: env.SETTLEMINT_MINIO_ENDPOINT!,
-      accessKey: env.SETTLEMINT_MINIO_ACCESS_KEY!,
-      secretKey: env.SETTLEMINT_MINIO_SECRET_KEY!,
-    });
+import { Client } from "minio";
+import type { Client as MinioClient } from "minio";
```

Comment:
the test is not using the "@settlemint/sdk-minio", use the library. Goals of the test is not to test minio but to test the code the sdk exposes